src/js/visual/configurator-csr.js

```diff
@@ -897,14 +870,19 @@ ripe.ConfiguratorCsr.prototype._initCsrRenderedInitials = async function() {
 };
 
 /**
- * Cleanups everything related to the CSR initials.
+ * Cleanups CSR initials.
  *
  * @private
  */
 ripe.ConfiguratorCsr.prototype._deinitCsrRenderedInitials = function() {
     // cleanup handlers
     this._unregisterInitialsHandlers();
+};
 
+/**
+ * Free resources used by the csr initials.
+ */
+ripe.ConfiguratorCsr.prototype._unloadCsrRenderedInitialsResources = function() {

```

Comment:
I'd simplify to just this... what do you think?
I'd do this for the others with Resources sufix
```suggestion
ripe.ConfiguratorCsr.prototype._unloadCsrRenderedInitials = function() {

```

Reply:
If you check the whole deinit logic it has like three methods. Each with a specific tas Ex for scene:

`_deinitScene()` - Destroys scene without disposing it's resources
`_unloadSceneResources()` - Disposes resources related to the scene, as in textures, meshes, materials, ... (it's basically cleanup of resources that the GC doesn't handle).
`_destroyScene`- Uses the other two methods. It completly destroys the scene and everything related to it.

So, if we want to change the names, I would keep something that indicates that there is a difference between them, that's why I added the "resources" suffix